### PR TITLE
sidebar-organism

### PR DESCRIPTION
**Pull request by PDhanrajnath**

### CODE REVIEW

File: src/components/Molecules/Sidebar/Sidebar.types.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export type SidebarProps = {
```
Comment (rohithZE) [inline src/components/Molecules/Sidebar/Sidebar.types.tsx:1]: It is better to create a file called `types` and add all types there

Comment (rohithZE) [reply src/components/Molecules/Sidebar/Sidebar.types.tsx:1]: The number of files will go up a lot if we keep adding types file for each component

File: src/components/Molecules/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -54,7 +48,7 @@ const Sidebar: React.FC<SidebarProps> = ({
   };
 
   return (
-    <div className={classes.root} onClick={clickHandler}>
+    <div className={class1} onClick={clickHandler}>
```
Comment (rohithZE) [inline src/components/Molecules/Sidebar/Sidebar.tsx:51]: Better naming here.
`stylesFromParent` or something like that
class1 does not indicate much right

Comment (PDhanrajnath) [reply src/components/Molecules/Sidebar/Sidebar.tsx:51]: ok Rohith
I'll change that

File: src/components/Molecules/Sidebar/Sidebar.types.tsx (hunk 1 above)
Comment (PDhanrajnath) [reply src/components/Molecules/Sidebar/Sidebar.types.tsx:1]: for all props, I need to have only one types.tsx file right?

File: src/components/types/myTypes.types.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export type SidebarProps = {
```
Comment (rohithZE) [inline src/components/types/myTypes.types.tsx:1]: types.tsx should be good enough as name

**Review by rohithZE — APPROVED** [review]

**Pull request merged**